Scripts/Editor/Build/AndroidKeystoreHelper.cs

```diff
@@ -0,0 +1,208 @@
+                   && !string.IsNullOrEmpty(PlayerSettings.Android.keyaliasPass);
+        }
+
+
+        [MenuItem(MENU_PATH_PREFIX+"/Set Keystore Passwords")]
+        private static void SetKeystorePasswords()
+        {
+            Logger.Debug("Starting...");
+            bool result = ApplyKeystorePasswordsFromUser();
+            Logger.Debug($"Complete. WasSuccessful:{result}");
+        }
+
+        [MenuItem(MENU_PATH_PREFIX+"/Clear Keystore Passwords")]
+        private static void ClearKeystorePasswordsFromPrefs()
+        {
+            Logger.Debug("Starting...");
+            EditorPrefs.DeleteKey(KeystorePassPrefsKey);
+            EditorPrefs.DeleteKey(KeystoreAliasPassPrefsKey);
+            Logger.Debug("Complete. Keys: (see below)" +
+                         $"\n {KeystorePassPrefsKey}" +
+                         $"\n {KeystoreAliasPassPrefsKey}");
+
+            EditorUtility.DisplayDialog(
+                "Keystore Passwords Deleted",
+                "Keystore passwords have been deleted from prefs.",
+                "Ok");
+        }
+
+        private static bool ApplyKeystorePasswordsFromPrefs()
+        {
+            PlayerSettings.Android.keystorePass = EditorPrefs.GetString(KeystorePassPrefsKey);
+            PlayerSettings.Android.keyaliasPass = EditorPrefs.GetString(KeystoreAliasPassPrefsKey);
+
+            Logger.Debug($"Keystore passwords retrieved from prefs. WasSuccessful:{AreKeystorePasswordsSet}, Keys:(see below)" +
+                         $"\n {KeystorePassPrefsKey}" +
+                         $"\n {KeystoreAliasPassPrefsKey}");
+            return AreKeystorePasswordsSet;
+        }
+
+        private static bool ApplyKeystorePasswordsFromUser()
+        {
+            EditorInputDialogResult keystorePass = EditorInputDialog.Show("Enter Keystore Password",
+                "Please enter the keystore password.",
+                "Keystore Password",
+                "Ok",
+                "Cancel");
+
+            if (keystorePass.Result == EditorInputDialogResult.ResultAction.Cancel)
+            {
+                Logger.Debug("Cancelled");
+                return false;
+            }
+
+            EditorInputDialogResult keystoreAliasPass = EditorInputDialog.Show("Enter Keystore Alias Pass",
+                "Please enter the keystore alias password.",
+                "Keystore Alias Password",
+                "Ok",
+                "Cancel");
+
+            if (keystoreAliasPass.Result == EditorInputDialogResult.ResultAction.Cancel)
+            {
+                Logger.Debug("Cancelled");
+                return false;
+            }
+
+            bool saveToPrefsResult = EditorUtility.DisplayDialog(
+                "Save to Editor Prefs?",
+                "Unity forgets these passwords each time the editor is quit." +
+                "\n\nWould you like to save them to your editor prefs and automatically enter them during future sessions?",
+                "Yes!",
```

Comment:
This exclamation mark is crucial and appreciated

Reply:
A little more serious than "Thanks Boss" but not much :)